pasqal_cloud/batch.py

```diff
@@ -51,7 +51,7 @@ class Batch(BaseModel):
     device_type: str
     project_id: str
     id: str
-    user_id: int
+    user_id: str
```

Comment:
This the only user_id field to maybe be mistyped? I looked briefly but didn't see

Reply:
Yep looked like it